{{cookiecutter.project_slug}}/.github/workflows/linting.yml

```diff
@@ -39,10 +39,9 @@ jobs:
           npm install
       - name: Run ESLint
         working-directory: ./client
-        {% if cookiecutter.client_app == 'Vue3' %}
+        {% if cookiecutter.client_app == 'Vue3' -%}
         run: npx eslint --ext .js,.vue src
-        {% elif cookiecutter.client_app == 'React' %}
+        {% elif cookiecutter.client_app == 'React' -%}
         run: npx eslint --ext .ts,tsx,js src
-        {% endif %}
-  {% endif %}
-
+        {% endif -%}
+{%- endif %}
```

Comment:
Running the project for the first time in a while.
There were a number of small regressions introduced in recent PRs that added extra whitespace and broke linters